entries/git-workflow-guidelines/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Git Workflow Guidelines"
+date: last-modified
+---
+
+{{< include /includes/_wip.qmd >}}
+
+### Introduction
+
+Note: In this and the following Git workflow posts, we assume that you know the basics of what Git is. If this is not the case, the Git-scm website includes great documentation, videos to get you started, as well as cheat sheets. (from VS code post)
```

Comment:
Maybe a link to the Git+scm website?  Also, did you mean to leave the comment in () in there?